tests/unittests/client/test_experiment_client.py

```diff
@@ -476,10 +476,12 @@ class TestBroken:
     def test_broken_trial(self):
         """Test that broken trials are detected"""
         with create_experiment(config, base_trial) as (cfg, experiment, client):
-            trial = client.suggest()
-            assert trial.status == "reserved"
-
-            atexit._run_exitfuncs()
```

Comment:
All other atexit in the tests need to be removed and replace with the with syntax.